CASSANDRA.md

```diff
@@ -153,9 +148,14 @@ object Phantom extends App {
 }
 ```
 
-Phantom requires mapping classes to lift the database model to DSL types. The query definition also requires special equality operators.
+Phantom requires mapping classes to lift the database model to DSL types. The query definition also requires special equality operators. The database model however adds fully type-safe schema compatible operations at compile time, meaning
+phantom will not allow you to produce invalid queries with respect to the Cassandra schema.
+
+As opposed to Quill and the Java Driver, phantom is schema aware, whereas the other two tools simply generate
```

Comment:
Could you remove this statement? Quill does have a feature to check the generated queries against the database.